package/models.py

```diff
@@ -3,7 +3,23 @@
 import subprocess
 import sys
 
-def display(snippet_name, password, clipboard=None):
+def display(snippet_name=None, password, clipboard=None):
```

Comment:
You can't place an argument with default value before an argument with no default values.. Suggesting you few changes, use any one of them.

Either simply revert changes:
```suggestion
def display(snippet_name, password, clipboard=None):
```

OR Do it like: 

```suggestion
def display(snippet_name=None, password=None, clipboard=None):
```